ui/src/views/Documentation/Reference/Entry.jsx

```diff
@@ -105,6 +105,8 @@ export default class Entry extends Component {
       this.props.entry.name === window.location.hash.slice(1) ||
       this.props.entry.type === window.location.hash.slice(1) ||
       encodeURIComponent(path(['content', '$id'], this.props.entry)) ===
```

Comment:
Ahh, It turns out we forgot to change `content` to `schema` in https://github.com/taskcluster/taskcluster/commit/a3367f8238ff4ef5eee006ff99c236109db740f7. I think that should fix the issue since `content.$id` has the same value as a`$id`. Good job for finding the issue!

Reply:
Thankyou . Debugging this was a good way out to understand the code